### DISCUSSION TITLE
[MWS] [Missing Orders in reports]

### DISCUSSION THREAD

**Discussion by graedwin** [amzn/selling-partner-api-models#3068] [Announcements]
Hello!

We are experiencing  issues as some orders are not getting included in the Reports due to an outage yesterday. You can retry requesting a new report so the data will be updated. We are currently investigating and will post updates as soon as possible. 

Javier V
Amazon Site Reliability Engineer

**Comment by moltar** [comment]
Is this just for MWS or SP API too?

**Comment by estebanangel** [comment]
is this issue happening again for the days between January 1 and January 7? have been retrieving data for those days and it's consistently lower than the one appearing in seller central
